Tella/Scenes/Settings/Views/Servers/ServerSelectionView.swift

```diff
@@ -48,6 +50,14 @@ struct ServerSelectionView: View {
                                      isValid: .constant(true), action: {
                 selectedServerType = .uwazi
             }).padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+            TellaButtonView<AnyView>(title: "GOOGLE DRIVE",
```

Comment:
What do you think if we craete a model with the attributes `Title`, ServerConnectionType  and add an array of this model with the different connections

Reply:
and for the action we can create a switch case ?